sockeye/layers.py

```diff
@@ -456,7 +456,7 @@ def hybrid_forward(self, F,
             updated_keys = F.concat(previous_keys, keys, dim=1)
             keys = _remove_first_step(F, updated_keys)
 
-        updated_values = keys
+        updated_values = values
```

Comment:
this bug didn't actually cause any harm because `previous_values` is not `None` in practice.